src/layers/core/grid-cell-layer/grid-cell-layer.js

```diff
@@ -66,15 +66,20 @@ export default class GridCellLayer extends Layer {
 
   getShaders() {
     return enable64bitSupport(this.props) ? {
-      vs: gridCellVertex64, fs: gridCellFragment, modules: ['fp64', 'project64', 'lighting']
+      vs: gridCellVertex64,
```

Comment:
As a personal preference, I wish we didn't have to expand into multiple lines - I really try to make the layer code have fewer lines. How about:

```
getShaders() {
      const {shaderCache} = this.context;
      return enable64bitSupport(this.props) ? {
 -      vs: vertex64, fs: fragment, modules: ['fp64', 'project64', 'lighting'], shaderCache
     } : {